src/tiled/tileseteditor.cpp

```diff
@@ -196,7 +196,13 @@ void TilesetEditor::addDocument(Document *document)
 
     TilesetView *view = new TilesetView(mWidgetStack);
     view->setTilesetDocument(tilesetDocument);
-    view->setZoomable(new Zoomable(this));
+
+    auto zoomable = new Zoomable(this);
+    auto fileName = tilesetDocument->fileName();
+    qreal zoom = Preferences::instance()->TilesetScale(fileName);
```

Comment:
Ok, just one more question, every tileset in docker should has the same zoom that was stored or only first opened?